host2str.c

```diff
@@ -1598,6 +1622,9 @@ ldns_rdf2buffer_str_svcparams(ldns_buffer *output, const ldns_rdf *rdf)
 		case LDNS_SVCPARAM_KEY_IPV6HINT:
 			st = svcparam_ipv6hint2buffer_str(output, val_sz, dp);
 			break;
+		case LDNS_SVCPARAM_KEY_DOHPATH:
+			st = svcparam_dohpath2buffer_str(output, val_sz, dp);
+			break;
```

Comment:
Since the function was no different than `svcparam_value2buffer_str()` and that is the `default:`, this whole `case` can be removed too.

```suggestion
```